warehouse/warehouse.go

```diff
@@ -105,6 +105,20 @@ const (
 	triggerUploadQPName = "triggerUpload"
 )
 
+var whDestNameMap = map[string]string{
+	"BQ":             "bigquery",
+	"RS":             "redshift",
+	"MSSQL":          "mssql",
+	"POSTGRES":       "postgres",
+	"SNOWFLAKE":      "snowflake",
+	"CLICKHOUSE":     "clickhouse",
+	"DELTALAKE":      "deletalake",
```

Comment:
Typo deletalake

Reply:
Also, Can we move this to warehouse.utils and Keys can be replaced with Constants.